kANe/v2p1/01_Topics_of_dharmashAstra.md

```diff
@@ -1,266 +1,177 @@
-+++
-title = "01 Topics of Dharmaśāstra"
++++ title = "01 Topics of Dharmaśāstra"
```

Comment:
Please revert this line.

Reply:
In the editor it is like
<img width="835" alt="First line" src="https://user-images.githubusercontent.com/29619813/191049936-b1e63e7c-ff5f-43c2-b449-ba14479968ae.PNG">

Reply:
Try committing and then pushing.